### DISCUSSION TITLE
Female Wizard/Barbarian heroes apperance

### DISCUSSION THREAD

**Discussion by a1exsh** [ihhub/fheroes2#4351] [Ideas]
I always found it amusing that the female Wizard heroes like Kalindra and Myra are still displayed as white-bearded old men, and that both on the adventure map and in combat. ;)

If we ever find a good pixel artist, maybe we should try to change that.

And after writing this I've also realized that this also applies to Barbarians Jezebel and Jaclyn...

**Comment by Branikolog** [comment]
> And after writing this I've also realized that this also applies to Barbarians Jezebel and Jaclyn...

The same to Knights and all other fractions.
It would be nice if we have female sprites for heroes. :)

**Reply by a1exsh** [reply]
I thought Knights are always fully armored, so it doesn't make such a big difference. The same is for Necromancer — the cloacked figure doesn't look wrong for a female hero.

Sorceress are always female and Warlocks are only males (at least the standard heroes). Or am I missing something more? :)

**Reply by Branikolog** [reply]
I think, for Knights and Necromancers we can create not a completely new sprite, but can change some accessories on a sprite to be distinctive from male heroes.  Like more elegant gloves for necromancer and different type of plume for knights.
